packages/webcomponents/src/styles/root.css

```diff
@@ -22,15 +22,15 @@
   --bs-gray-600: #6c757d;
   --bs-gray-700: #495057;
   --bs-gray-800: #343a40;
-  --bs-gray-900: #212529;
+  --bs-gray-900: var(--bs-body-color);
```

Comment:
In a follow up PR we should make sure that everything has bootstrap classes and we shouldn't need this file anymore, which will be awesome!